gettsim/parameters/soz_vers_beitr.yaml

```diff
@@ -189,32 +208,34 @@ soz_vers_beitr:
       standard: 0.01025
   2015-01-01:
     ges_krankenv:
-      ag: 0.073
-      an: 0.082
+      allgemein: 0.146
+      ermäßigt: 0.14
+      zusatz: 0.009
     ges_pflegev:
       standard: 0.01175
       zusatz_kinderlos: 0.0025
     arbeitsl_v: 0.015
     ges_rentenv: 0.0935
   2016-01-01:
     deviation_from: previous
-    note: Bekanntmachung Bundesanzeiger 30.12.2015 und  http://www.bmg.bund.de/ministerium/meldungen/2015/gkv-zusatzbeitrag.html
+    note: Bekanntmachung Bundesanzeiger 30.12.2015
     ges_krankenv:
-      an: 0.084
+      zusatz: 0.011
```

Comment:
Is this the sum of Sonderbeitrag and `mean_zusatzbeitrag`? If yes, we should separate them as one is the same for everybody and the other one is not.